templates/admin/org/list.tmpl

```diff
@@ -15,7 +15,6 @@
 					<table class="ui very basic striped table">
 						<thead>
 							<tr>
-								<th>ID</th>
```

Comment:
ok, but trust me, when you have more than 129 users, ID is just an annoyance and take a lot of place because the website CSS theme is not fluid.